soap-connect/integration-tests/pom.xml

```diff
@@ -14,6 +14,7 @@
     <packaging>jar</packaging>
 
     <properties>
+        <cxfVersion>3.0.13</cxfVersion>
```

Comment:
and shouldn't we changed that one? it does look like it doesn't make sense anymore.... All the components using CXF are now isolated and using different versions. Keeping that property is very error prone